resources/build/minimum-versions.inc.sh

```diff
@@ -0,0 +1,23 @@
+# Required minimum versions (also used as default version)
+# Minimum versions as of Keyman 18
+# (https://docs.google.com/document/d/1Uy3U2YXeA4rCEbUbT7O6QzUGJDZBeViFecL8fjUOLkE/edit?usp=sharing)
+
+# shellcheck shell=bash disable=SC2034 # SC2034: X appears unused.
+
+MIN_NODE_MAJOR_VERSION=18
+MIN_EMSCRIPTEN_VERSION=3.1.44    # 3.1.45 has problems, newer versions will work
+
+MIN_VISUAL_STUDIO_VERSION=2019
+MIN_MESON_VERSION=1.0.0
+MIN_CHROME_VERSION=95.0
+
+MIN_ANDROID_VERSION=5
```

Comment:
Are these variables intended to be used in build processes later? Or just for human-readable documentation?

fwiw, the Android build files use Sdk numbers that correspond to Android versions
e.g. Android 5.0 = SDK 21
Android 14.0 = SDK 34

https://github.com/keymanapp/keyman/blob/dfc3f63475cf53303eb1d843cfe6f8216a1eb694/android/KMAPro/kMAPro/build.gradle#L24

Reply:
> Are these variables intended to be used in build processes later? Or just for human-readable documentation?

Intent is for them to be used where possible in build processes. The idea is that this file is human readable also, but for developer documentation, not end users.

> fwiw, the Android build files use Sdk numbers that correspond to Android versions
> e.g. Android 5.0 = SDK 21
> Android 14.0 = SDK 34

It might be helpful to capture those version numbers, even if just in a comment for MIN_ANDROID_VERSION. It may not be possible to use these values directly in Gradle for example.

Reply:
```suggestion
MIN_ANDROID_VERSION=5
MIN_ANDROID_SDK_VERSION=21
```